doc/ipython-notebooks/clustering/KMeans.ipynb

```diff
@@ -288,7 +288,7 @@
     "new_initial_centers = array([[5.,5.],[0.,100.]])\n",
     "\n",
     "# set new initial centers\n",
-    "kmeans.put(\"mus\", new_initial_centers)"
+    "kmeans.put(\"cluster_centers\", new_initial_centers)"
```

Comment:
@vigsterkr I think there should be some sort of check when using put here. Also this should be done with a .put(“initial_cluster_centers”) and .put(“cluster_centers”) should be READONLY/watch_method.

Reply:
yep second that!